docs/data/charts/bars/bars.md

```diff
@@ -61,3 +61,22 @@ Bar charts can be rendered with a horizontal layout by providing the `layout="ho
 If you're using [composition](/x/react-charts/#multiple-charts), you should set the property `layout: 'horizontal'` to each bar series object.
 
 {{"demo": "HorizontalBars.js"}}
+
+## Animation
+
+To skip animation at the creation and update of your chart you can use the `skipAnimation` prop.
+When set to `true` it skips animation powered by `@react-spring/web`.
+
+Charts containers already use the `useReduceMotion` from `@react-spring/web` to skip animation [according to user preferences](https://react-spring.dev/docs/utilities/use-reduced-motion#why-is-it-important).
```

Comment:
Maybe also mention, that when using independent chart components, it's up to the user to call this hook? 🤔

Reply:
This hook is called in the ChartContainer so except if they create ther own charts container they have nothing to do. 

But to do that, they will have to know very well the codebase 😅